boards/m4a-wroom/doc.txt

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_m4a-wroom
+ * @ingroup     boards
+ * @brief       Mesh4All 2.4 GHz board.
```

Comment:
something like Mesh4all esp32 based board seems more appropriate

Reply:
do you refer to @brief?

Reply:
yes, i mean the brief

Reply:
Completed!